docs/ReleaseNotes.md

```diff
@@ -1,30 +1,34 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
-## Acuminator 3.1.2
+## Acuminator 3.1.2 - 2023-06-06
 Acuminator 3.1.2 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
 
 ### Enhancements
 In Acuminator 3.1.2, the following enhancements have been implemented:
- - The [PX1096](diagnostics/PX1096.md) diagnostic has been added to check signature of methods with the `PXOverride` attribute.
+ - The [PX1096](diagnostics/PX1096.md) diagnostic has been added to check signature of methods with the `PXOverride` attribute: The diagnostic checks that the base graph or base graph extensions contains a base method that is compatible with the method with the `PXOverride` attribute. 
  - The [PX1007](diagnostics/PX1007.md) diagnostic no longer checks the following DAC fields for a code annotation:
    - `Selected`
    - `Attributes`
    - `GroupMask`
    - `CompanyID`
    - `CompanyMask`
    - `DeletedDatabaseRecord`
- - The [PX1007](diagnostics/PX1007.md) diagnostic now checks that mapped DAC fields of projection DACs have the `inheritoc` XML tag
+ - The [PX1007](diagnostics/PX1007.md) diagnostic has been updated in the following ways:
+   - The diagnostic now supports the `inheritdoc` tag for DACs, DAC extensions, and their properties. The diagnostic also checks that mapped DAC field properties have the `inheritdoc` tag with the `cref` attribute pointing to the mapped DAC field property of the original DAC. 
+   - A new code fix can generate the `inheritdoc` tag for a mapped DAC property of a projection DAC.
```

Comment:
It's worth mentioning that VS allows us to apply code fix quickly to multiple alerts with its "Fix All" functionality for code fix.

Reply:
I have mentioned it in the diagnosic description., so added just a small mention here